File: left/pyramid.f90
program pyramid
    implicit none
    integer :: num, i, j
    character(len=100) :: block

    print *, 'Number:'
    read *, num

    do i = 1, num
        block = ''
        do j = 1, i
            block(j:j) = '*'
        end do
        print *, trim(block(1:i))
    end do
end program pyramid

File: left/pyramid.go
//Golang
package main
import "fmt"

func main() {
    var num int
    var block string
    
    fmt.Println("Number: ")
    fmt.Scanln(&num)    
    for i := 1; i < num+1; i++ {
        block += "*"
        fmt.Println(block)
    }
}
